extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetInstructions.java

```diff
@@ -33,7 +32,17 @@
  */
 public abstract class ParquetInstructions implements ColumnToCodecMappings {
 
-    private static volatile String defaultCompressionCodecName = CompressionCodecName.SNAPPY.toString();
+    public static final String DEFAULT_COMPRESSION_CODEC_NAME = "SNAPPY";
```

Comment:
Why don't we like the previous version?

Reply:
It was not showing up properly in the javadocs (although there isn't any javadoc for it right now, but just in spirit), and seemed like an unnecessary operation since users are also expected to set up strings.

Reply:
What I like about the current approach is compile-time linkage to the correct value (which is just an enum constant's `name()`/`toString()`).
You can certainly link to that enum constant in a JavaDoc, but maybe not the string itself. I'm not sure if that matters a lot.
Either way is probably fine, I just prefer to avoid "magic" constants when I have a way to source the values.

Reply:
Makes sense, reverted back to original approach.